src/tools/updateDeliveryData.js

```diff
@@ -0,0 +1,92 @@
+import { Op } from 'sequelize';
+import { ActivityReport } from '../models';
+import { auditLogger } from '../logger';
+
+const DELIVERY_DICTIONARY = {
+  Email: {
+    deliveryMethod: 'Virtual',
+    virtualDeliveryType: '',
```

Comment:
In the form if a user selects `Virtual` we make them select a delivery type. Do we need to add additional `deliveryType`s to the form so we can set this field to something?

Reply:
spoke to angela today - there isn't enough info to add delivery types to these reports